host-bin/enter-chroot

```diff
@@ -338,6 +338,15 @@ tmpfsmount() {
     mount -i -t tmpfs -o "rw${2:+,}$2" tmpfs "$target"
 }
 
+# Determines currently-logged in user
+getactiveuser() {
+    local activeuser="Default"
+    if [ -x "/usr/bin/jq" ]; then
```

Comment:
I like that you check for the existence of jq (in case it disappears in the future).  Is it possible, though, to do this with awk in a reasonable way?

Reply:
I don't know enough awk to answer this :blush: .. I do think jq will be around for a while since the config file is json.